backend/database/migrations/2022_11_24_164952_create_stacks_table.php

```diff
@@ -16,6 +16,7 @@ public function up()
         Schema::create('stacks', function (Blueprint $table) {
             $table->uuid('id')->unique();
             $table->string('name');
+            $table->uuid('user_id')->unique();
```

Comment:
what is happening here?  and if you want to add any column to a table, use the proper alter migration command to add and migrate

Reply:
Okay. I'm on it